templates/hoover-deps.nomad

```diff
@@ -351,6 +351,68 @@ job "hoover-deps" {
     }
   }
 
+  group "thumbnail-service" {
+
+    ${ continuous_reschedule() }
+    ${ group_disk() }
+
+    task "thumbnail" {
+      ${ task_logs() }
+      user = "root"
+
+      driver = "docker"
+      config {
+        image = "${config.image('thumbnail-service')}"
+        port_map {
+          thumbnail = 8000
+        }
+        labels {
+          liquid_task = "hoover-thumbnail"
+        }
+        memory_hard_limit = ${4 * config.thumbnail_memory_limit}
+        mounts = [ 
+          {
+            type = "tmpfs"
+            target = "/tmp"
+            readonly = false
+            tmpfs_options {
+              # set size here if you want
+            }
+          }
+        ]
+      }
+
+      resources {
+        memory = ${config.thumbnail_memory_limit}
+        cpu = 500
+        network {
+          port "thumbnail" {}
+          mbits = 1
+        }
+      }
+
+      service {
+        name = "hoover-thumbnail"
+        port = "thumbnail"
+        tags = ["fabio-/_thumbnail strip=/_thumbnail"]
+        check {
+          name = "http"
+          initial_status = "critical"
+          type = "http"
+          path = "/"
+          interval = "${check_interval}"
+          timeout = "${check_timeout}"
+        }
+        check_restart {
+          limit = 5
+          grace = "480s"
```

Comment:
Did you ever see it fail like this? This is needed when the container somehow enters a failure state without the container dying. For example, Tika and Rocketchat both have this behavior where they error out and start rejecting all HTTP traffic, but the containers are still up.

Let's leave out the `check_restart` from here unless you've seen it fail without the container dying.